canonical-web-and-design/maas-ui
# Pass on betterer failures

## Done

- Pass on betterer failures, and upload a build artifact of betterer results.

## QA

- Visit the check tab, and ensure a betterer results artifact has been uploaded.
